src/EnergyPlus/WeatherManager.cc

```diff
@@ -5596,6 +5599,16 @@ namespace WeatherManager {
                 ErrorsFound = true;
             }
 
+            // A9,  \field First Hour Interpolation Starting Values
```

Comment:
The new RunPeriod field is processed. It should be fail-safe in that by default or empty, it will use the setting of Hr24 (the old setting). Hr1 would be used with an explicit specification.